GrandCommonDivisor/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using GrandCommonDivisor.Core;
+using GrandCommonDivisor.Core.models;
+
+namespace GrandCommonDivisor.ConsoleUI
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            var isFirstValid = false;
+            var first = new NumberParseResult();
+
+
+            while (isFirstValid == false)
```

Comment:
I think you can create a private method for that and the next while loops, code is almost the same for both of them

Reply:
Fixed.